tests/dialects/test_cf.py

```diff
@@ -0,0 +1,28 @@
+from xdsl.ir import Block, OpResult
+from xdsl.dialects.arith import Addi, Subi, Muli, Constant
+from xdsl.dialects.builtin import i32
+from xdsl.dialects.cf import Branch, ConditionalBranch
+
+
+def test_branch():
+    block0 = Block()
+    _ = Branch.get(block0)
+
+
+def test_condbranch():
+    a = Constant.from_int_and_width(1, i32)
+    b = Constant.from_int_and_width(2, i32)
+    # Operation to add these constants
+    c = Addi.get(a, b)
+    d = Subi.get(a, b)
+    e = Muli.get(a, b)
+
+    block0 = Block.from_arg_types([i32])
+    block1 = Block.from_arg_types([i32])
+
+    branch0 = ConditionalBranch.get(c, block0, [d], block1, [e])
+    assert type(branch0.then) is OpResult
```

Comment:
I would probably replace that with `assert branch0.then is c` or something similar, to ensure the order in the constructor is correct as well!

Reply:
Sure!